.github/workflows/gem_deploy_ouroborus.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Ouroborus Gem
+
+on:
+  push:
+    tags:
+      - 'ouroborus*'
+
+jobs:
+  publish-gem:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.0 
+
+      - name: Publish Ouroborus Gem to GitHub Registry
+        run: |
+          mkdir -p ~/.gem
+          echo ":github: Bearer ${{ secrets.GITHUB_TOKEN }}" > ~/.gem/credentials
+          chmod 0600 ~/.gem/credentials
+          cd ouroborus
+          gem build ouroborus.gemspec
+          gem push --key github --host https://rubygems.pkg.github.com/geosales-evolution ouroborus-*.gem
```

Comment:
What does this '*'  in `ouroborus-*.gem` does ?

Reply:
wildcard * is used to match any name of gem, since in the current dir where gem push is called will be only one gem; it will be pushing the newly created gem.